src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -569,6 +570,16 @@ public void selectQuestionNumber(int qnNumber, int newQnNumber) {
         selectDropdownByVisibleValue(qnNumSelect, String.valueOf(newQnNumber));
     }
 
+    public boolean isSelectQuestionNumberEnabled(int qnNumber) {
+        WebElement qnNumSelect = browser.driver.findElement(By.id("questionnum-" + qnNumber));
```

Comment:
I didn't understand what you are trying to suggest :sweat_smile: . If you are suggesting to create a method like -
``` java
public WebElement getSelectQuestionNumberDropdown(int qnNumber) {
    return browser.driver.findElement(By.id("questionnum-" + qnNumber));
}
```

and use it in other methods? Well, it would only help `selectQuestionNumber()` and `isSelectQuestionNumberEnabled()` because `getSelectedQuestionNumber()` needs a `Select` object not a `WebElement` object. Or perhaps you mean something else?